global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "9.0.100-alpha.1.24058.9"
+    "version": "9.0.100-alpha.1.24067.4"
   },
   "tools": {
-    "dotnet": "9.0.100-alpha.1.24058.9",
+    "dotnet": "9.0.100-alpha.1.24067.4",
```

Comment:
@martincostello it works! Thanks so much!

Reply:
Awesome! I'll publish the 3.1.1 tag tomorrow.